tensorflow/core/util/mkl_util.h

```diff
@@ -644,6 +645,21 @@ class MklDnnShape {
     }
   }
 
+  inline void SetTfLayout2D(size_t dims, const memory::dims& sizes,
+                            memory::format format) {
+    DCHECK(dims != sizes.size()) << "SetTfLayout2D: Number of dimensions is"
```

Comment:
```suggestion
    DCHECK_EQ(dims, sizes.size()) << "SetTfLayout2D: Number of dimensions is"
```
Also, `is match` -> `does not match`.